packages/@aws-cdk/cli-lib-alpha/test/cli.test.ts

```diff
@@ -1,27 +1,18 @@
 import { join } from 'path';
-import * as core from 'aws-cdk-lib/core';
 import * as cli from 'aws-cdk/lib';
+import * as core from 'aws-cdk-lib';
```

Comment:
This is only importing `/core` on purpose.

Reply:
Because this doesn't seem to work in the new monorepo, I assume because `aws-cdk-lib` has been bundled.

Why must it import the submodule directly?

Reply:
Because it is best practice to import submodules and if that's not working anymore in the new monorepo that is extremely concerning to me. Especially if we don't understand why it is not working anymore.